chutney/packaging/local-dev/src/main/resources/application-sso-auth.yml

```diff
@@ -0,0 +1,45 @@
+#SPDX-FileCopyrightText: 2017-2024 Enedis
+#SPDX-License-Identifier: Apache-2.0
+
+spring:
+  security:
+    oauth2:
+      client:
+        registration:
+          my-provider:
+            provider: my-provider
+            client-id: my-client
+            client-secret: my-client-secret
+            authorization-grant-type: authorization_code
+            redirect-uri: "https://${server.http.interface}:${server.port}/login/oauth2/code/{registrationId}"
+            scope: openid, profile, email
+            client-name: My Provider
+        provider:
+          my-provider:
+            issuer-uri: http://localhost:3000
```

Comment:
Comment this and the backend will start without errors if the provider is not here.